neqo-client/src/main.rs

```diff
@@ -428,11 +428,17 @@ fn main() -> Res<()> {
 
     let mut args = Args::from_args();
 
+    let mut resumption_test = false;
```

Comment:
I would add an option to `Args` for this.  There is general utility in the resumption flag, even if it only works for the multi-connect option right now.  Having to pass separate flags is an awkwardness I'd prefer to avoid.

Reply:
I added "resume" option, but this option itself doesn't ensure that any additional connection will be made to obtain the token. So it should be combined with "download-in-series" option.

It's enabled implicitly when running "zerortt" or "resumption" test in QNS mode.